sig/datadog/appsec/configuration/settings.rbs

```diff
@@ -17,7 +17,7 @@ module Datadog
 
         def self.add_settings!: (untyped base) -> untyped
 
-        def self.enabled:  -> bool
+        def enabled:  -> bool
```

Comment:
This doesn't look correct :thinking:. Including this module in a class does not seem to cause a class to have a `#enabled` method?

Reply:
I think this is a case of https://github.com/soutaro/steep/issues/1232.